docs/contribute.rst

```diff
@@ -562,17 +562,22 @@ Release
 
 Follow the steps below to create a new major or minor release:
 
-* Use ``git log --oneline --no-decorate x.y-1..`` to generate the changelog
+
 * Update ``overview.rst`` with new contributors since the last release
 * Review the release notes in ``releases.rst``, update as needed
-* Add a ``Release tmt-x.y.0`` commit with the specfile update
+* Add a ``Release x.y.z`` commit, empty if needed: ``git commit --allow-empty -m "Release x.y.z"``
```

Comment:
How do we deal with cases where there are no new contributors and release notes don't require any changes? 
I'm thinking it might be easiest to just use the same workflow, with an empty commit, wdyt?

Reply:
Most likely there wouldn't be a situation like this. The only case I can think of is downstream rebuilds, but that is handled by downstream automation